CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+####		    ####
+#### Golpe Borghese ####
+####		    ####
+
+#Ban the PCI
+country_event = {
+  	id = 8020406
+  	title = "The Status of Opposition Parties"
+  	desc = EVT_8020406_DESC
+  	picture = "PCI_1970s"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1970 
+		NOT = { year = 1992 }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = nationalist
+		
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 1 }
+	
+	immediate = { clr_country_flag = ITA_years_of_lead remove_country_modifier = years_of_lead }
+	
+	option = {
+		name = "Outlaw the PCI, the PSI and lesser parties"
+		badboy = 3
+		
+		any_pop = {  
+		    limit = { is_primary_culture = yes }
+			ideology = { factor = 0.35 value = nationalist }
+			ideology = { factor = 0.10 value = conservative }
+		}
+		any_pop = {  
+		    scaled_consciousness = { ideology = progressive factor = 9 }
+			scaled_militancy = { ideology = progressive factor = 9 }
+			scaled_consciousness = { ideology = socialist factor = 9 }
+			scaled_militancy = { ideology = socialist factor = 9 }
+			scaled_consciousness = { ideology = communist factor = 9 }
+			scaled_militancy = { ideology = communist factor = 9 }
+			scaled_consciousness = { ideology = communist_social factor = 9 }
+			scaled_militancy = { ideology = communist_social factor = 9 }
+		}
+		
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 government = presidential_dictatorship } is_greater_power = yes }
+			diplomatic_influence  = { who = THIS value = 200 }
+			relation = { who = THIS value = 100 }
+        }
+		
+		add_country_modifier = { name = dirty_war duration = -1 }	
+		ai_chance = { factor = 0.80 }
+	}
+
+	option = {
+		name = "Outlaw the PCI and lesser parties"
+		badboy = 1
+		
+		any_pop = {  
+		    limit = { is_primary_culture = yes }
+			ideology = { factor = 0.25 value = nationalist }
+			ideology = { factor = 0.05 value = conservative }
+		}
+		any_pop = {  
+		    scaled_consciousness = { ideology = progressive factor = 4 }
+			scaled_militancy = { ideology = progressive factor = 4 }
+			scaled_consciousness = { ideology = socialist factor = 4 }
+			scaled_militancy = { ideology = socialist factor = 4 }
+			scaled_consciousness = { ideology = communist factor = 9 }
+			scaled_militancy = { ideology = communist factor = 9 }
+			scaled_consciousness = { ideology = communist_social factor = 9 }
+			scaled_militancy = { ideology = communist_social factor = 9 }
+		}
+		
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 government = presidential_dictatorship } is_greater_power = yes }
+			diplomatic_influence  = { who = THIS value = 200 }
+			relation = { who = THIS value = 50 }
+        }
+		
+		add_country_modifier = { name = dirty_war duration = -1 }
```

Comment:
This modifier should only last for 10 years if there is no catch all clean up event.